internal/game/game.go

```diff
@@ -288,29 +288,23 @@ func (board *BoardState) MovePawnAndSave(serializedMoveList string) error {
 	return nil
 }
 
-// Count pawns of the provided player that are in the provided target area.
-func (board BoardState) CountPawnsInTargetArea(player Player, targetArea [][]Cell) int8 {
-	pawns := int8(0)
+// Count pawns of each player that are in the player target area.
+func (board BoardState) CountPawnsInTargetAreas() (int8, int8) {
```

Comment:
It would be interesting to name arguments here as both are int8 types

```suggestion
func (board BoardState) CountPawnsInTargetAreas() (greenPawns int8, redPawns int8) {
```